e3nn/o3/_irreps.py

```diff
@@ -14,17 +15,14 @@ class Irrep(tuple):
     This class does not contain any data, it is a structure that describe the representation.
     It is typically used as argument of other classes of the library to define the input and output representations of functions.
 
-    Attributes
+    Parameters
     ----------
     l : int
-        non negative integer, the degree of the representation, :math:`l = 0, 1, \dots`
+        non-negative integer, the degree of the representation, :math:`l = 0, 1, \dots`
 
     p : {1, -1}
         the parity of the representation
 
-    dim : int
```

Comment:
This was missing a "Parameters" section, so I renamed this. I added a docstring to the properties, instead.